generatebundlefile/ecr_helper.go

```diff
@@ -73,13 +73,19 @@ func splitECRName(s string) (string, string, error) {
 		if len(chartNameList) == 4 {
 			return fmt.Sprintf("%s/%s", chartNameList[2], chartNameList[3]), chartNameList[3], nil
 		}
```

Comment:
I wonder if ==4 and > 4 scenario can be consolidated to keep the logic clean? I don't see issues with it but let me know if I omit something.